pinot-core/src/main/java/org/apache/pinot/core/util/AvroUtils.java

```diff
@@ -296,4 +302,33 @@ public static boolean isSingleValueField(Field field) {
       return fieldSchema;
     }
   }
+
+  public static void extractField(FieldSpec incomingFieldSpec, GenericRecord from, GenericRow to) {
+    String fieldName = incomingFieldSpec.getName();
+    //Handle MAP types
+    if (fieldName.toUpperCase().endsWith("__KEYS")) {
```

Comment:
Define __KEYS and __VALUES in CommonConstants.

Reply:
I will define them within the same class. this is very specific to the ingestion layer and we don't need it anywhere else.